packages/opentelemetry-rca-metrics/README.md

```diff
@@ -5,7 +5,10 @@
 [![devDependencies][devDependencies-image]][devDependencies-url]
 [![Apache License][license-image]][license-url]
 
-This module provides automatic collection of RCA Metrics
+This module provides automatic collection of RCA Metrics which includes basic metrics for:
```

Comment:
I still don't know what RCA stands for. Runtime [something] Analytics?

Reply:
I will change RCA everywhere for `Operation`

Reply:
btw RCA - root cause analysis, but the term "Operation" has been already suggested to me by someone else and I think it will be easier to understand, unless someone else have a better name for that ?

Reply:
I noticed that the Python project calls these "system metrics", and go divides their system metrics package into two packages, "host metrics", and "runtime metrics". We should see if we can align our naming with other projects, whichever we decide.

Reply:
it will be "host" then

Reply:
I think root cause analysis implies some actual analysis of problems. To me, operation metrics sounds like it could include host metrics and runtime metrics. Since we're only capturing cpu, disk, and memory metrics, I think "Host Metrics" is a more accurate and descriptive term. If we add a runtime metrics package later, we could compose them both into an "Operations metrics" package if we want.